src/App.tsx

```
@@ -24,6 +31,23 @@ function App() {
     setComputerMove(null);
   };
 
+  const checkWinner = () => {
+    if (playersMove === computerMove) {
+      return TIE;
+    }
+
+    switch (playersMove) {
+      case 'rock':
+        return computerMove === 'scissors' ? YOU_WIN : YOU_LOSE;
+      case 'paper':
+        return computerMove === 'rock' ? YOU_WIN : YOU_LOSE;
+      case 'scissors':
+        return computerMove === 'paper' ? YOU_WIN : YOU_LOSE;
+      default:
+        return null;
```

This case should not be possible, and I think you could throw an exception instead to indicate that.